node-engine/guides/create-app.mdx

```diff
@@ -8,7 +8,7 @@ The best way to understand how a Headless CMS operates is to create a simple app
 
 ## Setup WordPress
 
-1. Make a WordPress site. You should have a site and an Atlas account through WPEngine in the [User Portal](https://my.wpengine.com). Navigate to the site environment you want to use for your backend, and click the WP Admin link at the top of the page. This will log you into WP Admin, where the setup for the framework plugin and WPGraphQL occurs.
+1. Make a WordPress site. Your WordPress site is seperate from the site created when you create your app. You should have a site and an Atlas account through WPEngine in the [User Portal](https://my.wpengine.com). Navigate to the site environment you want to use for your backend, and click the WP Admin link at the top of the page. This will log you into WP Admin, where the setup for the framework plugin and WPGraphQL occurs.
```

Comment:
```suggestion
1. Make a WordPress site. Your WordPress site is separate from the site created when you create your app. You should have a site and an Atlas account through WPEngine in the [User Portal](https://my.wpengine.com). Navigate to the site environment you want to use for your backend, and click the WP Admin link at the top of the page. This will log you into WP Admin, where the setup for the framework plugin and WPGraphQL occurs.
```